src/Raven.Client/Constants.cs

```diff
@@ -210,6 +210,8 @@ public class Versioning
                 public const string EnableVersioning = "Raven-Enable-Versioning";
 
                 public const string DisableVersioning = "Raven-Disable-Versioning";
+
+                public const string RevisionedDocument = "Raven-Revisioned-Document";
```

Comment:
Can't we use flags? https://github.com/ravendb/ravendb/blob/v4.0/src/Raven.Server/Documents/DocumentFlags.cs#L9